src/app/helpers/options.helper.ts

```diff
@@ -7,8 +7,17 @@ export function mapToOptions(map: Map<string, string>): Option[] {
   return Array.from(map.entries()).map(([value, label]) => ({ label, value }));
 }
 
-export function choicesToOptions(): OperatorFunction<Choices, Option[]> {
+/**
+ * Convert choices to options
+ * @param [swap] use when value of object matched to option value
+ * @default key of object matched to option value
+ * @returns Option[]
+ */
+export function choicesToOptions(swap = false): OperatorFunction<Choices, Option[]> {
```

Comment:
I'd prefer not to add this option, and to instead just do map in that one place where it's needed. This really feels like an exception.